packages/playwright-core/src/client/page.ts

```diff
@@ -681,6 +681,8 @@ export class Page extends ChannelOwner<channels.PageChannel> implements api.Page
   async pause() {
     if (require('inspector').url())
       return;
+    if (this._connection.isRemote())
+      return;
```

Comment:
Wouldn't any console.log about skipped execution be helpful in this case? Then the user will know that there are some unexpected pauses in the code.